### DISCUSSION TITLE
How to access transform matrix?

### DISCUSSION THREAD

**Discussion by danwbean** [cnr-isti-vclab/PyMeshLab#308] [General] [answered]
How can I access the transform matrix from a filter that rotates, translates or scales?

`result = ms.compute_matrix_by_principal_axis(pointsflag=False, freeze=False)`

Filters similar to above return _None_ rather that the matrix.

**Comment by danwbean** [comment, marked as answer]
I found it...    It is an attribute of the Mesh object.  The typo in **'trasform'** rather than **transform** tripped me up

`mesh.trasform_matrix()`

Docs:
_trasform_matrix(self: pmeshlab.Mesh) → numpy.ndarray[numpy.float64[4, 4]]
Returns the 4x4 trasform matrix associated to the mesh._

**Reply by alemuntoni** [reply]
Thanks for reporting this! I'll fix and it will be available in the next pymeshlab version!
